models/comments.js

```diff
@@ -28,6 +28,10 @@ const CommentSchema = new Schema(
         type: String, // array of ownerId
       },
     ],
+    vote: {
+      type: Number,
+      default: 0,
+    },
```

Comment:
Don't update files you aren't suppose to.

Reply:
ok